### ISSUE TITLE
How exactly do you set permissions for this? 

### ISSUE THREAD

**Issue by MomokaChan** [me4502/SlotMachines#14]
I've tried PermissionsEX but I have no idea what to do, its all extremely complicated. The tutorial for it is outdated as well. So if you could give me the PEX command that would be great. Thank you

**Comment by me4502** [comment]
I don't believe PermissionsEx is being developed anymore, I'd recommend using LuckPerms instead.

The permissions for this plugin are outlined on the Ore page, https://ore.spongepowered.org/me4502/SlotMachines

`slots.create` to create slot machines, and `slots.create.admin` to create admin slot machines.

**Comment by MomokaChan** [comment]
alright thank you, I switched over to Luckperms. And I created a group, joined it, and did the command /lp group VIP permission set slots.create however, when I build the slot machine and click the button, it does nothing.

**Comment by me4502** [comment]
Are there any errors? @MomokaChan If you don't have permission it should provide a no permission message. Could you screenshot what you're making and provide versions?
